fblldbbase.py

```diff
@@ -36,14 +36,27 @@ def run(self, arguments, option):
     pass
 
 
-def evaluateExpressionValue(expression, printErrors=True):
+def evaluateExpressionValueWithLanguage(expression, language, printErrors):
   # lldb.frame is supposed to contain the right frame, but it doesnt :/ so do the dance
   frame = lldb.debugger.GetSelectedTarget().GetProcess().GetSelectedThread().GetSelectedFrame()
-  value = frame.EvaluateExpression(expression)
+  expr_options = lldb.SBExpressionOptions()
+  expr_options.SetLanguage(language)  # requires lldb r210874 (2014-06-13) / Xcode 6
+  value = frame.EvaluateExpression(expression, expr_options)
   if printErrors and value.GetError() is not None and str(value.GetError()) != 'success':
     print value.GetError()
   return value
 
+def evaluateExpressionValueInFrameLanguage(expression, printErrors=True):
+  # lldb.frame is supposed to contain the right frame, but it doesnt :/ so do the dance
+  frame = lldb.debugger.GetSelectedTarget().GetProcess().GetSelectedThread().GetSelectedFrame()
+  language = frame.GetCompileUnit().GetLanguage()  # requires lldb r222189 (2014-11-17)
```

Comment:
Not sure, it's not yet in 6.1.1, so `evaluateExpressionValueInFrameLanguage()` is not actually called anywhere and is present for future reference. Hopefully LLDB in 6.2 beta has it, but I haven't verified it.

I'll try it once it becomes final.